src/ts/__tests__/plugin.spec.ts

```diff
@@ -1,9 +1,18 @@
 import { AllSelection, TextSelection } from "prosemirror-state";
 import { br, doc, p } from "prosemirror-test-builder";
+import { commands } from "state";
 
 import { createEditor } from "./helpers";
 
 describe("Invisibles plugin", () => {
+  it("should not render invisibles when the plugin is not active", () => {
+    const view = createEditor(doc(p("1 2 3 4")));
+
+    const elements = view.dom.querySelectorAll(".invisible--space");
+
+    expect(elements.length).toEqual(3);
```

Comment:
should this not equal 0?

Reply:
The test is not testing what it says it tests! Thanks for spotting – just corrected.